app/code/Magento/Sales/etc/adminhtml/system.xml

```diff
@@ -106,15 +106,15 @@
                     <comment>We'll use the default error above if you leave this empty.</comment>
                 </field>
             </group>
-            <group id="dashboard" translate="label,comment" sortOrder="60" showInDefault="1" showInWebsite="0" showInStore="0">
+            <group id="dashboard" translate="label comment" sortOrder="60" showInDefault="1" showInWebsite="0" showInStore="0">
```

Comment:
I would recommend removing the `comment` completely. We don't have this section within a scope of the current node. The same is actual for  "Orders Cron Settings" group as well

Reply:
Also, it's a good idea to add `comment` value to the child 'field' node